tests/pages/compareSystemParametersPage.js

```diff
@@ -17,9 +17,14 @@ module.exports = {
     },
 
     verifyMetricsExistence () {
-        for (var i in this.metrics) {
+        I.click("/html/body/grafana-app/div/div/div/react-container/div/div[2]/div/div[1]/div/div[1]/dashboard-submenu/div/div[5]/dash-links-container/dash-link[2]/div/a");
+        for(let i=0; i<300; i++)
+            I.pressKey("ArrowDown")
```

Comment:
Spacing is wrong, why use ArrowDown, when you have figured out the footer element